test/cosmoz-bottom-bar.html

```diff
@@ -263,7 +262,7 @@
 					bottomBar.active = false;
 				});
 
-				test('menu should not be visible on screen', function () {
+				test('menu should not be visible on screen', function (done) {
```

Comment:
Adding the `done` argument is correct, but it actually reveals the test is wrong. 
Maybe you can remove that change, and we will handle it in another commit that fixes the test at the same time.